addons/account/views/report_invoice.xml

```diff
@@ -19,28 +19,28 @@
                         <span t-field="o.number"/>
                     </h2>
 
-                    <div class="row col mt32 mb32">
-                        <div class="mr32" t-if="o.name" name="description">
+                    <div class="row mt32 mb32">
+                        <div class="mr32 col-3" t-if="o.name" name="description">
```

Comment:
maybe just use `col` as discussed, to avoid line break?
are all mr/mb still useful?

Reply:
actually it will be less nice if there are a small number of elements